meet-data/napf/0904/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+NAPF,2010-03-08,USA,OH,Columbus,Titan Pro Bench Bash
```

Comment:
The date was reported as `3/8/2009` in the results file.

Not sure about location validity.

Also IPF.